scripts/generate-railroad-diagram.mts

```diff
@@ -10,7 +10,7 @@
  */
 
 import { PliParserInstance } from '../packages/language/src/parser/parser';
-import { createSyntaxDiagramsCode } from 'chevrotain';
+import { createSyntaxDiagramsCode } from '../packages/language/node_modules/chevrotain';
```

Comment:
I left the chevrotain dependencies where they currently are. The script is just a script - this will work fine.